src/web/make-graph.rkt

```diff
@@ -153,7 +161,11 @@
 
       (section ([id "cost-accuracy"] [class "section"]
                 [data-benchmark-name ,(~a (test-name test))]
-                ,@(if target-cost `([data-target-cost ,(~a target-cost)]) '()))
+                ; TODO: Doesn't show all data points
+
+                ,@(for/list ([target-cost list-target-cost])
+                    `[data-target-cost ,(~a target-cost)]))
```

Comment:
Wait, this doesn't make sense. You can't add multiple `data` attributes with the same name and have something meaningful happen.